Investigate tokenizer discrepancy

### Bug description

Might be that the tests just need to be adjusted (was not captured by CI because the Llama 3.2 token hasn't been added yet):

==================================================================================================================================== short test summary info =====================================================================================================================================
FAILED tests/test_tokenizer.py::test_tokenizer_against_hf[Llama-3.2-1B] - assert [9906, 11, 13...20, 1296, ...] == [128000, 9906...315, 420, ...]
FAILED tests/test_tokenizer.py::test_tokenizer_against_hf[Llama-3.2-1B-Instruct] - assert [128000, 1280...71, 2675, ...] == [128000, 1280...007, 271, ...]
FAILED tests/test_tokenizer.py::test_tokenizer_against_hf[Llama-3.2-3B] - assert [9906, 11, 13...20, 1296, ...] == [128000, 9906...315, 420, ...]
FAILED tests/test_tokenizer.py::test_tokenizer_against_hf[Llama-3.2-3B-Instruct] - assert [128000, 1280...71, 2675, ...] == [128000, 1280...007, 271, ...]
FAILED tests/test_tokenizer.py::test_tokenizer_against_hf[Mistral-7B-v0.3] - AssertionError: <class 'transformers.models.llama.tokenization_llama_fast.LlamaTokenizerFast'>
FAILED tests/test_tokenizer.py::test_tokenizer_against_hf[Mistral-7B-Instruct-v0.3] - AssertionError: <class 'transformers.models.llama.tokenization_llama_fast.LlamaTokenizerFast'>
===================================================================================================================== 6 failed, 174 passed, 2 xfailed, 25 warnings in 42.48s =====================================================================================================================


### What operating system are you using?

Unknown

### LitGPT Version

``` 



```

--- a/litgpt/tokenizer.py
+++ b/litgpt/tokenizer.py
@@ -94,7 +94,7 @@ class Tokenizer:
             config = json.load(fp)
         # for LlaMA-3 tokenizer there is no `add_bos_token` at all and `tokenizer_class` is only
         # `PreTrainedTokenizerFast`
-        if checkpoint_dir.stem.startswith("Meta-Llama-3"):
+        if checkpoint_dir.stem.startswith(("Meta-Llama-3", "Llama-3")):
             return True
         if "add_bos_token" in config:
             return config["add_bos_token"]

